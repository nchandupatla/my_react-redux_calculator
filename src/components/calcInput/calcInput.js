import React from 'react';

import './calcInput.css';
import { Button } from 'react-mdl';

const calcInput = (props) => (
    <Button className="calc-button op-button mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect" 
           onClick={props.clicked}>
        {props.label}
    </Button>
);

export default calcInput;