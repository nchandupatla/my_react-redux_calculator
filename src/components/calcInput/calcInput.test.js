import React from 'react';
import renderer from 'react-test-renderer';
import calcInput from './calcInput';


it('renders a snapshot', () => {
    const tree = renderer.create(<calcInput/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  