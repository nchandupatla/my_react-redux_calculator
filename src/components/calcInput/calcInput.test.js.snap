exports[`test renders a snapshot 1`] = `
  <Button className="calc-button op-button mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect" 
           onClick={props.clicked}>
        {props.label}
    </Button>
`;